packages/nextjs/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@thunderstore/nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "repository": "https://github.com/thunderstore-io/thunderstore-ui/tree/master/packages/nextjs",
```

Comment:
From the [docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository):
> The URL should be a publicly available (perhaps read-only) url that can be handed directly to a VCS program without any modification. It should not be a url to an html project page that you put in your browser. It's for computers.

A relevant example:
```json
{
  "repository": {
    "type": "git",
    "url": "https://github.com/facebook/react.git",
    "directory": "packages/react-dom"
  }
}
```